content/lxc/getting-started.md

```diff
@@ -80,9 +80,10 @@ With that done, the last step is to create an LXC configuration file.
 
 Those values should match those found in /etc/subuid and /etc/subgid, the values above are those expected for the first user on a standard Ubuntu system.
 
-Running unprivileged containers as an unprivileged user only works if you delegate a cgroup in advance (the cgroup2 delegation model enforces this restriction, not liblxc). Use the following systemd command to delegate the cgroup:
-
-    systemd-run --unit=myshell --user --scope -p "Delegate=yes" lxc-start <container-name>
+Running unprivileged containers as an unprivileged user only works if you delegate a cgroup in advance (the cgroup2 delegation model enforces this restriction, not liblxc). Use a systemd command to delegate the cgroup; this part of the documention is missing here and elsewhere:
+- https://wiki.debian.org/LXC/CGroupV2
+- https://wiki.archlinux.org/title/Linux_Containers
+- https://fedoraproject.org/wiki/LXC
```

Comment:
We shouldn't update the documentation to say that content is missing in the docs, but instead we should add the missing content - which can be by adding a link, as you are doing.

```suggestion
Running unprivileged containers as an unprivileged user only works if you delegate a cgroup in advance (the cgroup2 delegation model enforces this restriction, not liblxc). Use a systemd command to delegate the cgroup. See the following links for instructions on how to do this on different distributions:
- [Debian](https://wiki.debian.org/LXC/CGroupV2#LXC_containers_started_by_non-root)
- [Arch Linux](https://wiki.archlinux.org/title/Linux_Containers#Enable_support_to_run_unprivileged_containers_(optional))
- [Fedora](https://fedoraproject.org/wiki/LXC)
```
I can't see the corresponding information in the Fedora docs though - is it in there?